documentation/modules/adding-virt-provider.adoc

```diff
@@ -7,7 +7,7 @@
 = Adding {a-virt} destination provider
 
 You can add {a-virt} destination provider to the {ocp} web console in addition to the default {virt} destination provider, which is the provider where you installed {project-short}.
-
+// Is this still accurate?
 .Prerequisites
```

Comment:
Please answer question in line 10...

Reply:
we're not supposed to require a service account token with cluster-admin privileges for the target provider but unless @qiyuann says they checked it (it's different than adding a provider with user that doesn't have cluster-admin privileges on the local OCP instance), I'd keep this part and the rest looks accurate

Reply:
cluster-admin privileges is not a must. 
A service account token with privileges similar to rules defined in mtv-operator(forklift.konveyor.io, authentication.k8s.io, template.openshift.io are not required) could make remote migration work.

Reply:
@ahadas @qiyuann I wasn't clear enough. I meant is the description in line 9 still correct? I am going to leave the prerequisites as is unless you tell me otherwise.

Reply:
yes, it is correct but this made us look at line 13 below - and according to @qiyuann 's feedback we can change it as we put it in #375 :  `You must have {a-virt} link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/authentication_and_authorization/using-service-accounts[service account token] with sufficient permissions on MTV resources (e.g., providers, plans) and Openshift Virtualization resources (e.g., VirtualMachine, DataVolume)`